main.py

```diff
@@ -1,10 +1,53 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        date = json.loads(f.read())
+
+    def create_race(date: dict) -> Race:
+        date = date.get("race")
+        if not Race.objects.filter(name=date.get("name")).exists():
+            Race.objects.create(
```

Comment:
You can `return` the output of `.create()` here: this way you'll avoid unnecessary `.get()`. Same in `create_guild`.